pow/service.go

```diff
@@ -238,11 +237,13 @@ func (pow *Service) GenerateBlock(minerAddr string) (*types.Block, error) {
 	txs := pow.txMemPool.GetTxsInPool()
 	sort.Slice(txs, func(i, j int) bool {
```

Comment:
May be a better solution.
```go
	isHighPriority := func(tx *Transaction) bool {
		if tx.IsIllegalTypeTx() || tx.IsInactiveArbitrators() ||
			tx.IsSideChainPowTx() tx.IsUpdateVersion() ||
			tx.IsActivateProducerTx() {
			return true
		}
		return false
	}

	sort.Slice(txs, func(i, j int) bool {
		if isHighPriority(txs[i]) {
			return true
		}
		if isHighPriority(txs[j]) {
			return false
		}
		return txs[i].FeePerKB > txs[j].FeePerKB
	})
```

Reply:
Yes, that's a good refactor practice, good job.